fe1-web/docs/README.md

```diff
@@ -62,14 +69,23 @@ Here's the annotated directory tree:
 │   │
 │   ├── redux                    # module dealing with the global configuration of the application state (Redux-based)
 │   │
-│   └── styles                   # stylesheets
+│   ├── styles                   # stylesheets
+│   │
+│   └── types                    # some generic types
 │
 ├── features                 # independent features in the system
 │
-│   ├── connect                  # feature dealing with LAO/server connection
+│   ├── digital cash             # feature dealing with digital cash
+│   │
+│   ├── events                   # feature dealing with events happening in a LAO
+│   │
+│   ├── evoting                  # feature dealing with E-Voting and Elections
+│   │
+│   ├── home                     # feature dealing with the app's home screen
```

Comment:
should we put them after the `lao`, which provides the general structure of a feature ?

Reply:
Sure we can! It's just that I opted for the same order as the one in the IDE, which is alphabetical. But I agree it would be clearer this way, so I'm going to change it.